docs/developer-guide/etendo-copilot/available-tools/openapi-tool.md

```diff
@@ -0,0 +1,100 @@
+    - Identify the products of the order. Use the endpoint to search by similarity to find the Product ID. 
+    - Read the information of the products. Use the endpoint to read the Product by ID to get all the information of the Product. The endpoint retrieves the the information of the Product, it will be needed to insert the lines of the order.
+    - Check the body of the request to insert the order header, and build the body with the information of the Business Partner. The body description returned by the API tool will help you indicating where to find the information needed. Dont use the example values! If the description says to use a Specific ID, use it. But if the description says to use a value from the Business Partner, use the information of the Business Partner, received from api calls.
+
+    - Insert the header of the order. The api call will return the info of the Order Header. Remember the Order Header ID, because you will need it to insert the lines of the order.
+
+    - Check the body of the request to insert the order lines, and build the body with the information of the Products. The body description returned by the API tool will help you indicating where to find the information needed. Dont use the example values! If the description says to use a Specific ID, use it. But if the description says to use a value from the Product, use the information of the Product, received from api calls. Remember use the Order Header ID to insert the lines of the order. This  insertion is needed to be done for each product of the order. If the request not include any product, you can ask for the information to the user. If not include the quantity of the product, you can ask for it.
+
+    - Insert the lines of the order. The api call will return the info of the Order Lines. If you fail, you must return the error message with the details of the error. Dont forget to execute the api calls.
+
+
+    The final return message must be a link to order header. "http://localhost:8080/etendo/?tabId=294&recordId={ORDER_HEADER_ID}". The full link, because i want to copy  & paste it.
+  ```
+
+!!! note
+    In the last paragraph of the prompt, the link "http://localhost:8080/etendo/?tabId=294&recordId={ORDER_HEADER_ID}" is a link for localhost, it is necessary to replace it with the real link of the Etendo system.
+
+### Other Configurations:
+- Its necessary to add the ```gradle.properties``` file with the following configuration:
+``` properties
+    ETENDO_HOST=http://localhost:8080/etendo
+```
+!!! warning
+    Replace http://localhost:8080/etendo with the real link of the Etendo system.
+
+
+## Functionality
+
+The combination of the prompt with the functional concepts, the tool that allows to read and analyze the OpenAPI Spec to see the available endpoints and the tool to make the API calls, allows the defined assistant to load the purchase and its lines.
+
+## How to integrate Copilot with other API's
```

Comment:
I would move this to the page on how to create a tool or a separate one and here I would just leave the link. Incidentally, make it a little more extensive and add explanations or code examples.